### ISSUE TITLE
[All platforms/TubePlayer] TextBox Placeholder is cut off

### ISSUE THREAD

**Issue by rajamatt** [unoplatform/Uno.Samples#727]
![image](https://github.com/unoplatform/Uno.Samples/assets/70542961/97c3a197-f11b-439a-bb5a-dcab61d23172)

**Comment by rajamatt** [comment]
Will try to repro this in a minimal sample app.

**Comment by rajamatt** [comment]
After testing in a new uno app, here's what I found:

For a default Material theme TextBox, the placeholder text is cut off when the height is set to anything lower than 62.

```csharp
new TextBox()
    .Height(61) 
    .PlaceholderText("Placeholder")
```

![image](https://github.com/unoplatform/Uno.Samples/assets/70542961/dfc1d737-d737-40d7-8f8b-8c526556f782)

```csharp
new TextBox()
    .Height(62)
    .PlaceholderText("Placeholder")
```

![image](https://github.com/unoplatform/Uno.Samples/assets/70542961/661d7161-4b7b-4800-a7ad-def896bb39f2)

**Comment by rajamatt** [comment]
@kazo0 is this a known thing with textboxes?

**Comment by kazo0** [comment]
@rajamatt yes I believe it's because we are hardcoding a height in the code there. We should keep it at the default height for now

**Comment by agneszitte** [comment]
> Let's just remove the Height for now, we can revisit once [unoplatform/Uno.Themes#1014](https://github.com/unoplatform/Uno.Themes/pull/1014) is merged now that it's unblocked

@kazo0 I am okay with removing the fixed height, it is making more sense also in my opinion. But @NVLudwig the generated code from the figma file would need to follow as well in addition to @rajamatt updating the workshop-related code for the documentation.

**Comment by rajamatt** [comment]
@agneszitte I updated the workshop docs in this PR: https://github.com/unoplatform/workshops/pull/164
And removed the height in the singleproject PR: https://github.com/unoplatform/Uno.Samples/pull/728

**Comment by NVLudwig** [comment]
**A brief history of TextBox**
1. Tube Player design was made to repro an external design. 
2. When we started we identified a limitation of Uno TextBox: it did not allow a textBox to not have a floating label, it have a hard coded min height and did not allow trailing Icon buttons.
3. It was agreed we would create code for a TextBox style to allow having not floating label and trailing icon buttons and no min height.
4. A corresponding Figma branch of the Uno Material toolkit was create, reviewed and approved by DEV and merged in with the understanding the code for this would follow and all would be awesome. The file was pushed to the community and has been live ever since.
5. Tube Player had to be shipped and the PR was going to be merged imminently, the PR was never merged and design was never retrofited as PR was always going to merge imminently.
6. So now we have a disconnect between the figma file and the capabilities of the Uno TextBox

**Comment by agneszitte** [comment]
> **A brief history of TextBox**
> 
> 1. Tube Player design was made to repro an external design.
> 2. When we started we identified a limitation of Uno TextBox: it did not allow a textBox to not have a floating label, it have a hard coded min height and did not allow trailing Icon buttons.
> 3. It was agreed we would create code for a TextBox style to allow having not floating label and trailing icon buttons and no min height.
> 4. A corresponding Figma branch of the Uno Material toolkit was create, reviewed and approved by DEV and merged in with the understanding the code for this would follow and all would be awesome. The file was pushed to the community and has been live ever since.
> 5. Tube Player had to be shipped and the PR was going to be merged imminently, the PR was never merged and design was never retrofited as PR was always going to merge imminently.
> 6. So now we have a disconnect between the figma file and the capabilities of the Uno TextBox

Thank you for the details @NVLudwig.

@kazo0 based on what was shared here, should we ask to update the figma file to remove the fixed height so the generated code follow the change for the workshop like @rajamatt already pushed a PR for it here: https://github.com/unoplatform/workshops/pull/164

**Comment by NVLudwig** [comment]
Update for @agneszitte I have hit two road blocks:

1. Internal: found a bug with the [rendering of the textbox](https://github.com/unoplatform/Uno.Figma/issues/2505), now it's outer edge getting clipped
2. External: I can't publish updates to Tube Player at the moment. Waiting on Figma support communication.

**Comment by kazo0** [comment]
This shouldn't block the single project work, created an issue here to continue discussion and work: 

https://github.com/unoplatform/Uno.Samples/issues/734
